libs/domain/site/src/services/site.providers.ts

```diff
@@ -98,18 +97,14 @@ export const siteProviders: Provider[] = [
   },
   localeHydration,
   currencyHydration,
-  ...(featureVersion >= '1.1'
-    ? [
-        {
-          provide: BreadcrumbService,
-          useClass: DefaultBreadcrumbService,
-        },
-        {
-          provide: FallbackBreadcrumbResolver,
-          useClass: DefaultFallbackBreadcrumbResolver,
-        },
-      ]
-    : []),
+  {
+    provide: BreadcrumbService,
+    useClass: DefaultBreadcrumbService,
+  },
+  {
+    provide: FallbackBreadcrumbResolver,
+    useClass: DefaultFallbackBreadcrumbResolver,
+  },
```

Comment:
cool, these services/resolvers just new features and will not break anything. only the usage will be considered a breaking change.

Reply:
Yup, and currently resolvers are used by breadcrumbs service, and the service itself by the breadcrumb component